### DISCUSSION TITLE
Welcome to briefing Discussions!

### DISCUSSION THREAD

**Discussion by holtwick** [holtwick/briefing#85] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by paulponraj** [comment]
I wish it creates a permanent link instead of generating random links. Do you have any idea about that?

**Reply by holtwick** [reply]
I could list the last rooms you have visited on that machine, would that be what you are thinking of? 

Very good idea, I made a ticket for it https://github.com/holtwick/briefing/issues/124

**Reply by holtwick** [reply]
I added the feature, it should be online now. See brie.fi/ Visit a room and get back to the home page to see the previous room listed.
